connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewClient.kt

```diff
@@ -0,0 +1,167 @@
+package com.stripe.android.connect.webview
+
+import android.annotation.SuppressLint
+import android.graphics.Bitmap
+import android.webkit.JavascriptInterface
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import com.stripe.android.connect.BuildConfig
+import com.stripe.android.connect.EmbeddedComponentManager
+import com.stripe.android.connect.PrivateBetaConnectSDK
+import com.stripe.android.core.Logger
+import com.stripe.android.core.version.StripeSdkVersion
+import kotlinx.coroutines.runBlocking
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+
+@OptIn(PrivateBetaConnectSDK::class)
+internal class StripeConnectWebViewClient(
+    private val connectComponent: StripeConnectURL.Component,
+    private val logger: Logger = Logger.getInstance(enableLogging = BuildConfig.DEBUG),
+    private val jsonSerializer: Json = Json {
+        ignoreUnknownKeys = true
+    }
+) : WebViewClient() {
+
+    override fun onPageStarted(view: WebView, url: String?, favicon: Bitmap?) {
+        initJavascriptBridge(view)
+    }
+
+    @SuppressLint("SetJavaScriptEnabled")
+    fun configureAndLoadWebView(webView: WebView) {
+        webView.apply {
+            webViewClient = this@StripeConnectWebViewClient
+            settings.apply {
+                javaScriptEnabled = true
+                domStorageEnabled = true
+                loadWithOverviewMode = true
+                useWideViewPort = true
+                userAgentString = "$userAgentString - stripe-android/${StripeSdkVersion.VERSION_NAME}"
+            }
+            addJavascriptInterface(StripeJsInterface(), ANDROID_JS_INTERFACE)
+            addJavascriptInterface(StripeJsInterfaceInternal(), ANDROID_JS_INTERNAL_INTERFACE)
+
+            val publishableKey = EmbeddedComponentManager.getInstance().configuration.publishableKey
+            loadUrl(StripeConnectURL.getStripeURL(connectComponent, publishableKey))
```

Comment:
should be able to do this after making the other changes
```suggestion
            loadUrl(EmbeddedComponentManager.getInstance().getStripeUrl(connectComponent))
```